### ISSUE TITLE
npm i -g  @subql/query error

### ISSUE THREAD

**Issue by chaosma** [OnFinality-io/subql#182]
I cannot install 

`npm i -g  @subql/query`

```
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.com/@subql%2fquery - Not found
npm ERR! 404
npm ERR! 404  '@subql/query@latest' is not in the npm registry.
```

But I can install cli and node.

**Comment by jiqiang90** [comment]
Hi, thank you for your feedback and sorry for the hassle.

We updated our document yesterday but haven't published the @suql/query to npm immediately.

Now we have just published v0.2.0, and It should be on NPM now. Please try to install it again, and when you run it also make sure your environment variables are correct. Thanks

`DB_HOST=localhost
subql-query --name <project_name> --playground`

**Comment by chaosma** [comment]
Thanks, it works now!
